cou/steps/analyze.py

```diff
@@ -240,41 +196,19 @@ def _get_minimum_cloud_series(self) -> Optional[str]:
         :rtype: Optional[str]
         """
         return min(
-            (app.series for app in self.apps_control_plane + self.apps_data_plane),
+            (app.series for app in self.apps),
             default=None,
         )
 
     @property
-    def data_plane_machines(self) -> dict[str, juju_utils.Machine]:
-        """Data-plane machines of the model.
-
-        :return: Data-plane machines of the model.
-        :rtype: dict[str, Machine]
-        """
-        return {
-            machine_id: app.machines[machine_id]
-            for app in self.apps_data_plane
-            for machine_id in app.machines
-        }
-
-    @property
-    def control_plane_machines(self) -> dict[str, juju_utils.Machine]:
-        """Control-plane machines of the model.
+    def machines(self) -> dict[str, juju_utils.Machine]:
+        """All OpenStack machines of the model.
 
-        :return: Control-plane machines of the model.
+        :return: All OpenStack machines of the model.
         :rtype: dict[str, Machine]
         """
         return {
             machine_id: app.machines[machine_id]
-            for app in self.apps_control_plane
+            for app in self.apps
             for machine_id in app.machines
         }
-
-    @property
-    def machines(self) -> dict[str, juju_utils.Machine]:
-        """All OpenStack machines of the model.
-
-        :return: All OpenStack machines of the model.
-        :rtype: dict[str, Machine]
-        """
-        return {**self.data_plane_machines, **self.control_plane_machines}
```

Comment:
Note that there's also some other refactoring along the way - I tend to do these kind of improvements while making other changes and trying to understand the code.  I can split them out of the PR once it's ready.